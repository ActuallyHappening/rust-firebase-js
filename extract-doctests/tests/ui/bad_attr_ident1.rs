use extract_doctests::extract_doctests;

#[extract_doctests(bad_attr(template = ""))]
fn placeholder() {}

fn main() {}
