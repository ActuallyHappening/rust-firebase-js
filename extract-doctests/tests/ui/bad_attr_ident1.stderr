error: Expected inline_config, not bad_attr
 --> tests/ui/bad_attr_ident1.rs:3:20
  |
3 | #[extract_doctests(bad_attr(template = ""))]
  |                    ^^^^^^^^
